Directory.Build.targets

```diff
@@ -1,4 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project>
   <Import Project="Sdk.targets" Sdk="Microsoft.DotNet.Arcade.Sdk" Condition="'$(ArcadeBuild)' == 'True'"/>
-</Project>
\ No newline at end of file
+
+  <PropertyGroup>
```

Comment:
These are product settings and as such should be in the `src/` directory not in the root.

Reply:
FWIW, CoreFX has these in `eng/` directory: https://github.com/dotnet/corefx/blob/master/eng/versioning.targets#L44

Reply:
Putting them in `eng/` is fine with me, just not the root.